lib/config/definitions.js

```diff
@@ -1281,6 +1281,20 @@ const options = [
     },
     mergeable: true,
   },
+  {
+    name: 'mix',
+    releaseStatus: 'alpha',
```

Comment:
I usually use "alpha" to mean "I know there are some things that don't work that need to be added". I think let's use "beta", which I use to mean "Ready for use on real repositories, but take care to check each PR just in case".